### ISSUE TITLE
rename directory only not works

### ISSUE THREAD

**Issue by looric** [neurobin/rnm#24]
rnm -rs '/\./-/gd' ./*

compiled from source under centos 7

  File: ‘/usr/bin/rnm’
  Size: 4404320         Blocks: 8608       IO Block: 4096   regular file
Device: f900h/63744d    Inode: 134304408   Links: 1
Access: (0755/-rwxr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
Access: 2019-07-17 13:24:51.269169462 +0800
Modify: 2019-07-17 13:23:49.309173870 +0800
Change: 2019-07-17 13:23:49.309173870 +0800

**Comment by looric** [comment]
Name         : rnm (Bulk rename utility)
Version      : 4.0.5

**Comment by looric** [comment]
```
[root@neptune rnm-4.0.5]# rnm -rs '/\./-/gd' ./*

/root/rnm/rnm-4.0.5/aclocal.m4    ---->    [...]/aclocal-m4

Select action:

    1. Confirm for this file only.
    2. Confirm for all.
    3. Skip this file.
    4. Skip all and exit
```

**Comment by looric** [comment]
Sorry , it works like expected.

**Comment by neurobin** [comment]
Actually I found that it does not work. somewhere in the update process this feature was broken. Fixed in 4.0.6. 

`rnm -rs '/\./-/gd' ./*` should apply the replacement on only directories.
